daemon/logger/factory.go

```diff
@@ -36,7 +36,7 @@ func (ctx *Context) Hostname() (string, error) {
 	return hostname, nil
 }
 
-// Command returns the command that the container being logged was started with
+// Command returns the command that the container being logged was started with. The Entrypoint is prepended to the container arguments.
```

Comment:
Not sure if there's a *rule* in the docker source code but I think it would be better to have this comment on multiple lines (with a max of ~80 char per line). but that is a tiny tiny nit :blush: